WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -489,12 +487,23 @@ private extension OrderListViewController {
     ///
     func createFilterConfig() ->  EmptyStateViewController.Config {
         guard let filters = viewModel.filters, filters.numberOfActiveFilters != 0 else {
-            return emptyStateConfig
+            return noOrdersAvailableConfig()
         }
 
         return noOrdersMatchFilterConfig()
     }
 
+    /// Creates EmptyStateViewController.Config when there are no orders available
+    ///
+    func noOrdersAvailableConfig() -> EmptyStateViewController.Config {
+        .simple(
+            message: NSAttributedString(string: Localization.allOrdersEmptyStateMessage),
+            image: .waitingForCustomersImage,
+            onPullToRefresh: { [weak self] refreshControl in
+                self?.pullToRefresh(sender: refreshControl)
+            })
+    }
+
     /// Creates EmptyStateViewController.Config for no orders matching the filter empty view
     ///
     func noOrdersMatchFilterConfig() -> EmptyStateViewController.Config {
```

Comment:
nit: do you think we could also apply the pull-to-refresh to this empty state when there are no orders matching the filters? like if there are new orders coming in, it'd be nice if the user can refresh the orders

Reply:
I agree. Added in https://github.com/woocommerce/woocommerce-ios/pull/7023/commits/81bbe9fbb587ba268a0c5fc25a77aa45d29c1a3a